test/box/graceful_shutdown.test.lua

```diff
@@ -0,0 +1,99 @@
+net_box = require('net.box')
+fiber = require('fiber')
+env = require('test_run')
+test_run = env.new()
+
+------------------------------------------------------------
+-- When receiving a shutdown signal server
+-- should stop accept new connections,
+-- close for read existing not graceful connections
+-- and do some work before the shutdown is complete
+------------------------------------------------------------
+
+--guest can have some of these privileges
+_, _ = pcall(function() box.schema.user.grant('guest','execute,write,read','universe') end)
+
+_ = box.schema.space.create('counter')
+_ = box.space.counter:create_index('primary')
+
+test_run:cmd("create server remote with script='box/proxy.lua'")
+test_run:cmd("start server remote")
+test_run:cmd("switch remote")
+_, _ = pcall(function() box.schema.user.grant('guest','execute,write,read','universe') end)
```

Comment:
you can add `box.schema.user.grant("guest", "read,write,execute,create,drop", "universe", nil, {if_not_exists = true})`
in corresponding *.lua file. Also you should create new graceful_shutdown.lua file, not used proxy.lua. Because i think that you should test new abiulity with several iproto thread. See `gh-5645-several-iproto-threads.lua` for example https://github.com/tarantool/tarantool/pull/6038/commits/6a472334fdea1d32ac4a1d8ae9ad60cea1294041

Reply:
Created `graceful_shutdown.lua` file and added several iproto threads to the `box.cfg`